### ISSUE TITLE
Testnet linking to another network with different genesis block

### ISSUE THREAD

**Issue by fix** [LiskHQ/lisk#89]
See the screenshots:
- first screenshot is the current testnet (0.2.0)
- second is old testnet, still running (0.1.4)

both are "abridged" with ip "185.86.79.245" (running 0.1.4) which is appearing in both network `/api/peers`

On a side note, just see how old network is connecting to the whole new network. (i guess because of seed peers configured in config.json)

<img width="1241" alt="screen shot 2016-05-04 at 10 45 39" src="https://cloud.githubusercontent.com/assets/709388/15009225/988b9044-11e5-11e6-8d49-35f1183a47f4.png">

----

<img width="1211" alt="screen shot 2016-05-04 at 10 50 28" src="https://cloud.githubusercontent.com/assets/709388/15009320/15677394-11e6-11e6-904d-ee2538596dd8.png">

**Comment by karmacoma** [comment]
> i guess because of seed peers configured in config.json

I deployed completely different seed peers with the release of 0.2.0, and didn't reuse the same seed peers.

Looking at the peers listing on my node, this peer connected to the 0.2.0 network, and has possibly reverted back to 0.1.4, so has announced itself to both networks.

```
{"ip":"185.86.79.245","port":7000,"state":1,"os":"linux3.13.0-85-generic","sharePort":1,"version":"0.2.0"},
```

Peers from both networks do speak with each other. According to the current implementation, they should not find any blockchain commonality, and will ban each other for 60 minutes at a time.

Looking at: https://testnet.lisk.io/api/peers there are no 0.1.4 peers in our db, this is as a result of this code:

- https://github.com/LiskHQ/lisk/blob/development/modules/transport.js#L95
- https://github.com/LiskHQ/lisk/blob/development/modules/transport.js#L580

**Comment by fix** [comment]
Oh, i see another possibility: I remember at some points we played with different versions in config.json. So maybe the owner changed manually the version in the config and then reverted back. But somehow the version is not updated.
On a side note, i don't think make a declarative field for the version is a good idea. It should not be configurable.

btw, what is the difference between login.lisk.io and testnet.lisk.io? is login.lisk.io supposed to become the mainnet?

**Comment by fix** [comment]
should be fixed with #132
